jwst/regtest/test_nirspec_ifu_spec2.py

```diff
@@ -54,3 +63,95 @@ def test_spec2(run_spec2, fitsdiff_default_kwargs, suffix):
     """Regression test matching output files"""
     rt.is_like_truth(run_spec2, fitsdiff_default_kwargs, suffix,
                      truth_path=TRUTH_PATH)
+
+
+@pytest.mark.bigdata
+def test_nirspec_ifu_user_supplied_flat(jail, rtdata_module, fitsdiff_default_kwargs):
```

Comment:
All the new tests in this module should be function-scoped too.  Remove `jail` and `rtdata_module`, replace with `rtdata`.